DerivedCategories/examples/julia/TiltingEquivalence.jl

```diff
@@ -1,133 +1,167 @@
 
 using HomalgProject
 
-SizeScreen( [ 1000, 100 ] )
-
 LoadPackage( "DerivedCategories" )
 
 ReadPackage( g"DerivedCategories", g"examples/pre_settings.g" ); GAP.Globals.ENABLE_COLORS = true
 
 ℚ = HomalgFieldOfRationals()
 
-q = RightQuiver( "q",
+q = RightQuiver( "q_𝓞",
           [ "𝓞(0)", "𝓞(1)", "𝓞(2)" ],
           [ "x0", "x1", "x2", "y0", "y1", "y2" ],
           [ 1, 1, 1, 2, 2, 2 ],
           [ 2, 2, 2, 3, 3, 3 ] )
+SetLabelsAsLaTeXStrings( q, julia_to_gap([ g"x_1", g"x_2", g"x_3", g"y_0", g"y_1", g"y_2" ]) )
```

Comment:
Only one comment, maybe setting these LaTeX attributes would be improved after merging https://github.com/homalg-project/Algebroids/pull/65 and requiring the latest version of Algebroids package.

Reply:
Now merged.

Reply:
Shall I update the PR?

Reply:
Yes please